apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/test/template-selector-control-test.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * External dependencies
+ */
+/* eslint-disable import/no-extraneous-dependencies */
+import { uniqueId, omit } from 'lodash';
+/* eslint-enable import/no-extraneous-dependencies */
+
+import { render, fireEvent } from '@testing-library/react';
+
+import { templatesFixture, blocksByTemplatesFixture } from './helpers/templates-blocks-helpers';
+import { TemplateSelectorControl } from '../template-selector-control';
+
+// Mock out this component until @wordpress/block-editor
+// `BlockPreview` component is available as default export.
+// Once available, swap this mock to mocking out `BlockPreview`
```

Comment:
Minor: If we haven't already it might be a good idea to add a ticket to the backlog so we don't forget about swapping this.

Reply:
Good point. https://github.com/orgs/Automattic/projects/116#card-26717544